pageserver/src/http/openapi_spec.yml

```diff
@@ -579,6 +579,12 @@ paths:
         required: false
         schema:
           type: integer
+      - name: lazy
```

Comment:
Should we put this into the `TenantLocationConfigRequest` instead?

Reply:
John on https://github.com/neondatabase/neon/issues/6315#issuecomment-1963711127 was thinking next to flush_ms, and I agree that's a nice place.